students/15th/team2/seunghee_mun/board/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from django.utils import timezone
+from user.models import User
+
+
+class Board(models.Model):
+    board_name = models.CharField(max_length=45)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    time = models.DateTimeField(default=timezone.now)
+    image = models.CharField(max_length=200, blank=True)
+    contents = models.TextField()
+
+    class Meta:
+        db_table = 'boards'
+
+class Comment(models.Model):
+    board_title = models.ForeignKey(Board, on_delete=models.CASCADE)
```

Comment:
ForeignKey를 담고있는 컬럼명은 참조하고 있는 테이블명의 단수형으로 지정해주시면 됩니다.
```suggestion
    board = models.ForeignKey(Board, on_delete=models.CASCADE)
```

Reply:
수정완료하였습니다.